docs/src/content/docs/datamessages.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: "Data Messages"
+description: Protocol documentation for messages exchanged between client and server during Ultravox calls.
+---
+
+Data messages are used to communicate state, transcripts, and control information between the client and server during Ultravox calls. These messages work across WebRTC data channels and WebSocket connections.
+
+## Message Structure
+
+All messages are JSON objects containing:
+- A required `type` field identifying the message type
+- Additional fields specific to each message type
+- Messages follow camelCase naming convention
+
+## Client Messages
+
+Messages sent from client to server:
+
+### Input Messages
+- **InputTextMessage**
```

Comment:
Might be useful to have a brief description of what each message is before going into its schema.  E.g.:

- **InputTextMessage** - A user message sent via text.
    + `type: "input_text_message"`
    + `text`: The content of the user message
    
I think I included docstrings for all the messages in data_messages.py that ought to give a good starting point.

Reply:
done.